### DISCUSSION TITLE
How to enable Google Adsense on a Quarto blog website?

### DISCUSSION THREAD

**Discussion by rohitfarmer** [quarto-dev/quarto-cli#2898] [Q&A]
Hi there, I am new to the Quarto website/blogs. I can activate/deploy Google Analytics, but I cannot find a way to allow Google Adsense on my website. Is there a way to do it? Thanks!

**Comment by dragonstyle** [comment]
We don't natively support google Adsense - you'll need to include the javascript and so on to activate that yourself.

**Reply by rohitfarmer** [reply]
Thank you for your reply. The first JavaScript that connects the site to Google Adsense needs to be inserted in the head section. How can I do that, so that when I render the website, it goes to the right place? I guess the scripts that place the ads can be inserted at desired positions within the post and should be straightforward.

**Reply by dragonstyle** [reply]
Check out this on HTML includes, which should make it pretty easy to target the header or after the body, should that be needed:

https://quarto.org/docs/output-formats/html-basics.html#includes

**Comment by coatless** [comment]
@rohitfarmer I recently made a Quarto extension for this as I work to port different website from `blogdown`/`hugo` over to `quarto`.  You might find it helpful:

<https://github.com/coatless-quarto/adsense>
